### DISCUSSION TITLE
Escape should unfocus the url bar

### DISCUSSION THREAD

**Discussion by Samueru-sama** [zen-browser/desktop#1142] [Ideas]
Right now zen has this firefox issue where if you press ecape when focused on the URL bar nothing happens, in firefox you are supposed to use F6 instead while chromium fixed this issue long ago.

The other solution would be to make that action a configurable keybind.

**Comment by linhub15** [comment]
This would be a nice feature to add, but it might be coming in Firefox 133.
- https://bugzilla.mozilla.org/show_bug.cgi?id=1086524

New preference will be added called `browser.urlbar.focusContentDocumentOnEsc`.
We should be able to access it through `about:config` once the version is released.

### Update
Just tested on the Firefox nightly version 133.0a1, works beautifully.

**Reply by Samueru-sama** [reply]
AFTER 10 FUCKING YEARS, FINALLY!

**Reply by Abdullah16M** [reply]
I vote for this to be the default in zen
